tests/gbs/test_vibronic.py

```diff
@@ -67,6 +71,22 @@ def test_duschinsky(self):
         Ud = np.diag(self.wp ** -0.5) @ J @ np.diag(self.w ** 0.5)
         assert np.allclose(Ud, self.Ud)
 
+    def test_invalid_temperature(self):
+        """Test if function raises a ``ValueError`` when a negative temperature is given."""
+        with pytest.raises(ValueError, match="Temperature must be zero or positive"):
+            vibronic.gbs_params(self.w, self.wp, self.Ud, self.d, -1)
+
+    def test_twomode(self):
+        """Test if function returns two-mode squeezing parameters that correctly reconstruct the
+        input normal mode frequencies."""
+        w = -k * self.T / (0.5 * h * c * 100) * np.log(np.tanh(self.t))
+        assert np.allclose(w, self.w)
+
+    def test_zero_temperature(self):
+        """Test if function returns zero two-mode squeezing parameters when temperature is zero."""
+        t, _, _, _, _ = vibronic.gbs_params(self.w, self.wp, self.Ud, self.d, 0.0)
+        assert np.all(t == 0)
```

Comment:
why do we need `np.all()` here?

Reply:
`t` is an N dimensional numpy array so `t==0` evaluates to something like `[True, True, False, True, True]`
The all acts on a list of bools and only returns true if all the items in the list are true

Reply:
because t is an array.

Reply:
My bad, thought `t` was a float